spec/factories/user_cycle_states.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :user_cycle_state do
+    association :user, factory: :user
+    association :cycle, factory: :cycle
+    repository_link { 'https://fakeurl.com' }
+    completed_at { '2022-11-05 19:47:04' }
+    reviewer_id { 2 }
```

Comment:
I think this line could throw an error in test env if there are not enough users